src/recommender.py

```diff
@@ -390,20 +392,27 @@ def call_insights_recommender(payload):
         try:
             # TODO remove hardcodedness for payloads with multiple ecosystems
             if payload and 'ecosystem' in payload[0]:
-
+                quickstarts = True
                 if payload[0]['ecosystem'] in RecommendationTask.chester_ecosystems:
                     INSIGHTS_SERVICE_HOST = os.getenv("CHESTER_SERVICE_HOST")
-                else:
-                    INSIGHTS_SERVICE_HOST = os.getenv("PGM_SERVICE_HOST") + "-" + \
-                                            payload[0]['ecosystem']
+                elif payload[0]['ecosystem'] == 'maven':
```

Comment:
I think this should be structured a bit differently, if ecosystem is in hpf_ecosystems then you should check for maven as a special case of it and decide whether kronos needs to be used.

Reply:
But in that case, we will have to put 2 checks instead of one(first for hpf-ecosystems and second for maven). And since the check happens only in case of Maven, it is easier to check for maven. Also, I have kept Kronos as default for the time being, since a majority of the request will be from OSIO.

Reply:
I see your point now. Working on the change.